### DISCUSSION TITLE
Survey Questions

### DISCUSSION THREAD

**Discussion by maxion1234** [Michaelrt86/ITSC-4155-Final-Project#6] [General]
Potential Survey Questions

**Comment by Michaelrt86** [comment]
What time would you like to have quiet hours?

**Reply by staghetti** [reply]
8pm
10pm
12am

**Comment by Michaelrt86** [comment]
What is your Major and do you want a roommate who has the same major as you?

**Reply by Michaelrt86** [reply]
------Text and Boolean

**Comment by Michaelrt86** [comment]
What time do you usually go to sleep?

**Reply by staghetti** [reply]
8-10
10-12
12-later

**Comment by Michaelrt86** [comment]
What are your study habits?

**Reply by Michaelrt86** [reply]
------Text Answer

**Comment by staghetti** [comment]
What are your hobbies?

**Reply by Michaelrt86** [reply]
------Text Answer

**Comment by staghetti** [comment]
How do you prefer your room climate?

**Reply by Michaelrt86** [reply]
--------Pick from Group String

**Comment by staghetti** [comment]
What year are you in?

**Reply by staghetti** [reply]
Freshman
Sophomore
Junior
Senior
Graduate

**Reply by Michaelrt86** [reply]
--------Pick from Group String

**Comment by staghetti** [comment]
How do you feel about sharing personal items (e.g., kitchen supplies, toiletries)?

**Reply by staghetti** [reply]
1-5

**Comment by staghetti** [comment]
How tidy do you typically keep your living space?

**Comment by staghetti** [comment]
How do you handle conflict when it arises with a roommate?
- Address it immediately
- Wait and see if it resolves itself
- Prefer to avoid it
